src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,241 @@
+
+## Step 1: Install the OpenTelemetry collector [#install-generic]
+
+To do a basic installation for single hosts in the cloud or on-premises, see [OpenTelemetry's instructions](https://opentelemetry.io/docs/collector/getting-started/#linux-packaging) for up-to-date installation steps from the community. Instructions are available for the following:
+
+* Linux: Debian systems
+* Linux: Red Hat
+* Windows
+* Docker, Kubernetes, and other options
+
+Your deployment experience might vary depending on which vendor-specific distributions you use. For example, installation via a package manager might be available for Linux hosts.
+
+<Callout variant="important">
+To set up infrastructure monitoring, you need to install and configuring components that are included in the `collector-contrib` release. For example, the host receiver is required to collect basic host metrics such as CPU, memory, disk and network stats and is only available in the [OpenTelemetry collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) release.
+</Callout>
+
+## Step 2: Configure the OpenTelemetry collector for New Relic [#configure]
+
+These examples are meant to serve as a starting point from which you can extend, customize, and validate configurations before using them in production.
+
+### Host monitoring using the host receiver [#host-receiver]
+
+The `collector-contrib` release provides a `hostreceiver` that generates metrics about the system scraped from various sources. It is intended to be used when the collector is deployed as an agent.
+
+When using the host receiver as part of the collector configuration, New Relic automatically detects host metrics as part of a `Host` entity and will synthesize its golden metrics providing the same experience as with the New Relic infrastructure agent. The following are the configuration requirements to enable the `Host` entity experience in New Relic UI:
+
+  * `host.id` attribute is present in host metrics.
+  * `service.name` and `container.id` attributes are not present in host metrics.
+
+Learn more about available metrics and advanced configurations from the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver).
+
+Adapt the `config.yaml` with these recommended parameters:
+
+<Callout variant="important">
+CPU, load, memory and disk utilization metrics require otelcol-contri release `v47` or greater.
+</Callout>
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "290px" }}>
+        Configuration
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        `receivers::hostmetrics`
+      </td>
+      <td>
+        Enable host metrics, 20 seconds interval is recommended (same default as the Infrastructure Agent), it should not be greater than 60 seconds to avoid issues with “host not responding” alerts. Process instrumentation is optional.
```

Comment:
infrastructure agent l/c
First sentence seems like it really ought to be two

Reply:
@barbnewrelic, I divided these up into a bullet list!